pytorch_lightning/trainer/progress.py

```diff
@@ -153,6 +153,9 @@ def load_state_dict(self, state_dict: dict) -> None:
         self.total.load_state_dict(state_dict["total"])
         self.current.load_state_dict(state_dict["current"])
 
+    def reset_on_restart(self):
+        self.current.reset_on_restart()
```

Comment:
maybe include these changes in the other PR that deals with the reset refactor?

Reply:
Yes, done !